tests/unit/frameworks/continuous_learning/multihead/multihead_test.py

```diff
@@ -28,18 +28,38 @@
 import unittest
 
 from torch.utils.data import DataLoader
+from torchvision import transforms
 
-from nupic.research.frameworks.continuous_learning.multihead.subdataset import (
-    get_datasets,
+from nupic.research.frameworks.continuous_learning.multihead.multihead import (
+    get_active_classes,
 )
+from nupic.research.frameworks.pytorch.dataset_utils.dataset_utils import split_dataset
+from nupic.research.frameworks.pytorch.test_utils.fake_data import (
+    FakeDataPredefinedTargets,
+)
+
+
+def setup(scenario):
+    fake_train_dataset = FakeDataPredefinedTargets(image_size=(1, 28, 28))
+    fake_test_dataset = FakeDataPredefinedTargets(image_size=(1, 28, 28))
+
+    if scenario in ("task", "domain"):
+        target_transform = transforms.Lambda(lambda y: y % 2)
+    else:
+        target_transform = None
 
+    train_datasets = split_dataset(fake_train_dataset,
+                                   groupby=lambda x: x[1] // 2,
+                                   target_transform=target_transform)
+    test_datasets = split_dataset(fake_test_dataset,
+                                  groupby=lambda x: x[1] // 2,
+                                  target_transform=target_transform)
 
-def setup(dataset_name, scenario):
-    train_datasets, test_datasets = get_datasets(dataset_name, scenario=scenario)
     train_loaders = [DataLoader(train_dataset, batch_size=256, shuffle=True)
                      for train_dataset in train_datasets]
-    test_loaders = [DataLoader(test_dataset, batch_size=1000, shuffle=True)
+    test_loaders = [DataLoader(test_dataset, batch_size=500, shuffle=True)
```

Comment:
Should use smaller batch sizes for unit tests. Unit test should not care about accuracy, the purpose of batch size in here should be the minimum batch size possible to cover the functionality being tested. 500 seems to large for that purpose.

Reply:
Ok, I'll change to 50 for tests.